tests/tasks/snowflake/test_snowflake.py

```diff
@@ -76,3 +76,46 @@ def test_execute_fetchall(self, monkeypatch):
         ).run()
 
         assert output == ["TESTDB"]
+
+
+class TestSnowflakeQueriesFromFile:
+    def test_construction(self):
+        task = SnowflakeQueriesFromFile(
+            account="test", user="test", password="test", warehouse="test"
+        )
+        assert task.autocommit is None
+
+    def test_file_path_must_be_provided(self):
+        task = SnowflakeQueriesFromFile(
+            account="test", user="test", password="test", warehouse="test"
+        )
+        with pytest.raises(ValueError, match="A file path must be provided"):
+            task.run()
+
+    def test_execute_error_must_pass_through(self, monkeypatch, tmpdir):
+        snowflake_module_connect_method = MagicMock()
+        connection = MagicMock(spec=sf.SnowflakeConnection)
+
+        # link all the mocks together appropriately
+        snowflake_module_connect_method.return_value = connection
```

Comment:
You can just define `snowflake_module_connect_method = MagicMock(return_value=connection)`